### PR TITLE
test cargo changes

### PR DESCRIPTION
**Pull request by gyakovlev**

### CODE REVIEW

File: eclass/cargo.eclass [hunk 1]
```diff
@@ -146,7 +146,7 @@ cargo_src_compile() {
 
 	export CARGO_HOME="${ECARGO_HOME}"
 
-	cargo build -j $(makeopts_jobs) $(usex debug "" --release) "$@" \
+	cargo -vv build -j $(makeopts_jobs) $(usex debug "" --release) "$@" \
```
Comment (stefson) [inline eclass/cargo.eclass]: can you imagine to make the -vv optional?

Comment (gyakovlev) [reply eclass/cargo.eclass]: yes, changes coming soon.
default will be verbose though.
but it'll be possible to override it back to quiet.
also NOCOLOR will be supported.

Comment (gyakovlev) [reply eclass/cargo.eclass]: https://github.com/gentoo/gentoo/pull/14135

**Comment by gyakovlev** [comment]
closing, took a bit different route

**Pull request closed without merging**